src/main/webapp/js/nav.js

```diff
@@ -0,0 +1,91 @@
+const loadingScreen = document.querySelector('.loading-screen')
+const mainNavigation = document.querySelector('.main-navigation')
+
+// Function to add and remove the page transition screen
+function pageTransitionIn() {
+    // GSAP methods can be chained and return directly a promise
+    // but here, a simple tween is enough
+    return gsap
+        .to(loadingScreen, { duration: .5, scaleY: 1, transformOrigin: 'bottom left' })
+}
+// Function to add and remove the page transition screen
+function pageTransitionOut(container) {
+    // GSAP methods can be chained and return directly a promise
+    return gsap
+        .timeline({ delay: 0.25 })
+        .add('start')
+        .to(loadingScreen, {
+            duration: 0.5,
+            scaleY: 0,
+            skewX: 0,
+            transformOrigin: 'top left',
+            ease: 'power1.out'
+        }, 'start')
+        .call(contentAnimation, [container], 'start')
+}
+
+// Function to animate the content of each page
+function contentAnimation(container) {
+    // Query from container
+    $(container.querySelector('.green-heading-bg')).addClass('show')
+    // GSAP methods can be chained and return directly a promise
+    return gsap
+        .timeline()
+        .from(container.querySelector('.is-animated'), {
+            duration: 0.5,
+            translateY: 10,
+            opacity: 0,
+            stagger: 0.4
+        })
+        .from(mainNavigation, { duration: .5, translateY: -10, opacity: 0 })
+}
+
+$(function() {
```

Comment:
we can use an arrow here

Reply:
could you please clarify? I'm unsure how to use an arrow here, I think I've only ever done it with .then() in promise chains

Reply:
this line can just be `$(() => {`

Reply:
done